support/developer/biztalk/management-operations/maintain-troubleshoot-database.md

```diff
@@ -31,56 +33,56 @@ Microsoft BizTalk Server databases and the health of the databases are important
 
 This article describes how to maintain BizTalk Server databases and how to troubleshoot BizTalk Server database issues.
 
-## You must disable the auto update statistics and auto create statistics options
+## You must disable the auto create statistics and auto update statistics options
 
 You must keep the `auto create statistics` and `auto update statistics` options disabled on the `BizTalkMsgBoxDb` database. To determine whether these settings are disabled, execute the following stored procedures in SQL Server:
 
 ```sql
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics'
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics'
 ```
 
-You should set the CurrentSetting setting to off. If this setting is set to on, turn it off by executing the following stored procedures in SQL Server:
+You should set the current setting to `OFF`. If this setting is set to `ON`, turn it off by executing the following stored procedures in SQL Server:
 
 ```sql
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics', 'off'
-exec sp_dboption 'BizTalkMsgBoxDB', 'auto update statistics', 'off'
+EXEC sp_dboption 'BizTalkMsgBoxDB', 'auto create statistics', 'off'
```

Comment:
For the sp "sp_dboption", it's no longer available in the latest versions of SQL Server, right? Do we need to update the commands by using [ALTER DATABASE SET options](https://learn.microsoft.com/en-us/sql/t-sql/statements/alter-database-transact-sql-set-options?view=sql-server-ver16#auto_create_statistics)?

Reply:
@aartig13, could you help answer this question?